src/main/java/duke/exception/InvalidCommandTimeException.java

```diff
@@ -0,0 +1,24 @@
+package duke.exception;
+
+public class InvalidCommandTimeException extends Exception {
```

Comment:
Good work here in creating new exception classes handling separate purposes.

Reply:
thank you!